src/RestClient.cs

```diff
@@ -134,6 +156,18 @@ private void HandleErrors(HttpWebResponse webResponse, string response)
             {
                 data = JsonConvert.DeserializeObject(response);
                 errorCode = data["error"]["code"];
+                if (host.Equals("AUTH"))
+                {
+                    if (statusCode >= 400 && statusCode < 500)
+                    {
+                        errorCode = ErrorCodes.BAD_REQUEST_ERROR.ToString();

```

Comment:
a more detailed error would be better here

Reply:
This is actually just a mapping of error code and not defining error message, that will be passed further as it is. The above change is due to the reason that the SDK currently is tightly coupled with what we have in API monolith. Since we are calling Auth service here, we dont receive such error codes in response. Hence we are just setting  `errorCode` based on `statusCode` we receive from APIs via auth service.